### PR TITLE
Add getting-started in language Spanish for LB4

### PR DESCRIPTION
**Pull request by HaroldV**
Hello

I created a getting-started in  spanish version of the document.
Would it be possible to merge this?

Thank you.

### CODE REVIEW

**Comment by HaroldV** [comment]
@nabdelgadir new pull request can you check it please ?

File: pages/es/lb4/iniciando.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+lang: es
+title: 'Iniciando'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar_es
```
Comment (nabdelgadir) [inline pages/es/lb4/iniciando.md]: `es_lb4_sidebar`

File: _data/sidebars/es_lb4_sidebar.yml [hunk 2]
```diff
@@ -13,9 +13,9 @@ translated: true
 children:
 
 - title: 'Iniciando'
-  url: Getting-started.html
+  url: iniciando.html
```
Comment (nabdelgadir) [inline _data/sidebars/es_lb4_sidebar.yml]: I think the url should stay as the English one. From [Translating articles](https://loopback.io/doc/en/contrib/Translating_articles.html#procedure):

> Change permalink from `/doc/en/lb2/Foo.html` to `/doc/xx/lb2/Foo.html`.



>

File: pages/es/lb4/iniciando.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+permalink: /doc/es/lb4/iniciando.html
```
Comment (nabdelgadir) [inline pages/es/lb4/iniciando.md]: Same here about the url

File: pages/es/lb4/iniciando.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
+---
+lang: es
```
Comment (nabdelgadir) [inline pages/es/lb4/iniciando.md]: Please add `layout: translation`

**Review by nabdelgadir — COMMENTED** [review]
Just a few comments 😄

**Comment by HaroldV** [comment]
Can you check ? @nabdelgadir

**Review by nabdelgadir — APPROVED** [review]
LGTM 👍Thanks!

**Pull request merged**